--- a/extras/package/macosx/package.mak	2013-03-14 10:56:15.000000000 +0100
+++ b/extras/package/macosx/package.mak	2014-04-17 13:47:18.674092532 +0200
@@ -36,7 +36,7 @@ VLC-tmp: vlc
 	mkdir -p $(abs_top_builddir)/tmp/extras/package/macosx/vlc.xcodeproj/
 	sed -e s,../../../contrib,$(CONTRIB_DIR),g $(srcdir)/extras/package/macosx/vlc.xcodeproj/project.pbxproj \
         > $(abs_top_builddir)/tmp/extras/package/macosx/vlc.xcodeproj/project.pbxproj
-	REVISION=`(git --git-dir=$(srcdir)/.git describe --always || echo exported)` && \
+	REVISION=exported && \
 	    sed "s/#REVISION#/$$REVISION/g" $(top_builddir)/extras/package/macosx/Info.plist \
         > $(top_builddir)/tmp/extras/package/macosx/Info.plist
 	cp -R $(top_builddir)/extras/package/macosx/Resources $(top_builddir)/tmp/extras/package/macosx/
